### PR TITLE
Music Synchronization

### PR DESCRIPTION
**Pull request by stefan-balke**
Hey, I added a music synchronization example and some audio files for it.

Furthermore, I added some more information to the readme.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2016, Brian McFee
+## ISC License
```
Comment (bmcfee) [inline LICENSE]: I'm not sure we can have / want a blanket license on this repo.  Each example might be licensed individually, and I'd like to allow some wiggle room there.  Similarly, as you noted below, the example tracks have different licenses from the code.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
-A collection of example notebooks demonstrating librosa functionality
+# Librosa Gallery
+
+A collection of code examples demonstrating some of librosa's functionality.
+
+## Examples
+
+
+## Notebooks
+
```
Comment (bmcfee) [inline README.md]: I keep the notebooks in this repo for reference and ease of development, but I think it's better to refer folks to the sphinx-gallery build https://librosa.github.io/librosa_gallery/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,23 @@
...
+* Audio Effects: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/02-Audio_effects.ipynb
+* HPRSS: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/02-HPRSS.ipynb
+* Vocal Separation: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/03-Vocal_separation.ipynb
+* Music Segmentation: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/04-Laplacian segmentation.ipynb
+* Superflux Onsets: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/05-Superflux_Onsets.ipynb
+* Rhythm Analysis: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/06-Rhythm_Analysis_Mellin_Transform.ipynb
+* Enhanced Chroma: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/07-Enhanced_Chroma.ipynb
+* Overriding Presets: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/08-Overriding_Defaults_with_Presets.ipynb
+* Music Synchronization: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/09-Music_Synchronization.ipynb
```
Comment (bmcfee) [inline README.md]: Why the file renames?  I'm fine with it, but wondering if it's just to make it easier for markdown links?

**Comment by bmcfee** [comment]
Awesome, thanks!

If the example is finished, can you also render it down to python in sphinx-gallery example format, as done in the `examples/` directory?  The doc builder doesn't parse notebooks properly, so the final versions all live as plain python with structured comments.

File: README.md (hunk 3 above)
Comment (stefan-balke) [reply README.md]: Just wanted to have consistency...

**Comment by stefan-balke** [comment]
Oh, I forgot about the gallery things. Stupid question: Why don't we simply do a simple jupyter html export?

Any idea on how to integrate the `display.Audio(data=x_1, rate=fs)` to the sphinx-gallery?

**Comment by stefan-balke** [comment]
And then link from http://librosa.github.io directly to the generated html files.

**Comment by bmcfee** [comment]
> Oh, I forgot about the gallery things. Stupid question: Why don't we simply do a simple jupyter html export?
> 

The short version is that sphinx-gallery looks nice and makes it easier to index (top-level thumbnails, for example).  It also does intersphinx cross-referencing automatically, so when you use librosa functions in the code, you can click on the code and go directly to the docs.

> Any idea on how to integrate the display.Audio(data=x_1, rate=fs) to the sphinx-gallery?

Can't do it, sorry.  It's probably not a great idea anyway, since it would be a big fat uncompressed audio buffer kicking around in unicode strings.

**Comment by stefan-balke** [comment]
> Can't do it, sorry. It's probably not a great idea anyway, since it would be a big fat uncompressed audio buffer kicking around in unicode strings.

I was thinking of an HTML audio-tag rather than pasting all the binary data to the file.

Alright, I'll port it and then see if I can fiddle around with the audio data.

How can build it locally? `make html` in the docs folder?

**Comment by bmcfee** [comment]
> How can build it locally? make html in the docs folder?

Yup.  You'll need the gallery extension for sphinx, but otherwise it ought to build just like any other sphinx project.

**Comment by stefan-balke** [comment]
Alright, I think I'm done here. It builds locally and looks reasonable. :)

File: docs/conf.py [hunk 4]
```diff
@@ -56,7 +56,7 @@
 
 # General information about the project.
 project = 'librosa-gallery'
-copyright = '2016, librosa development team'
+copyright = '2017, librosa development team'
```
Comment (bmcfee) [inline docs/conf.py]: please revert the copyright date.  It holds from the start of copyright, not the date of latest revision.

Comment (stefan-balke) [reply docs/conf.py]: done.

**Comment by bmcfee** [comment]
Thanks!

**Pull request merged**